sdk/datashare/Azure.ResourceManager.DataShare/tests/TestCase/SynchronizationSettingTests.cs

```diff
@@ -39,15 +39,14 @@ private async Task<DataShareSynchronizationSettingCollection> GetHronizationSett
 
         [TestCase]
         [RecordedTest]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/36058")]
         public async Task HronizationSettingApiTests()
         {
             //1.CreateorUpdate
             var collection = await GetHronizationSettingCollectionAsync();
             var name = Recording.GenerateAssetName("TestHronization-");
             var name2 = Recording.GenerateAssetName("TestHronization-");
             var name3 = Recording.GenerateAssetName("TestHronization-");
-            var input = ResourceDataHelpers.GetSynchronizationData();
+            var input = ResourceDataHelpers.GetSynchronizationData(Recording.UtcNow);
```

Comment:
instead of using data related with real life, could we just hard code a date here? For instance the date of today.

Reply:
fixed